utils/hdparm/patches/010-ldflags.patch

```diff
@@ -0,0 +1,13 @@
+--- a/Makefile
++++ b/Makefile
+@@ -13,9 +13,8 @@ oldmandir = $(manprefix)/man
+ CC ?= gcc
+ STRIP ?= strip
+ 
+-CFLAGS := -O2 -W -Wall -Wbad-function-cast -Wcast-align -Wpointer-arith -Wcast-qual -Wshadow -Wstrict-prototypes -Wmissing-prototypes -Wmissing-declarations -fkeep-inline-functions -Wwrite-strings -Waggregate-return -Wnested-externs -Wtrigraphs $(CFLAGS)
++CFLAGS ?= -O2 -W -Wall -Wbad-function-cast -Wcast-align -Wpointer-arith -Wcast-qual -Wshadow -Wstrict-prototypes -Wmissing-prototypes -Wmissing-declarations -fkeep-inline-functions -Wwrite-strings -Waggregate-return -Wnested-externs -Wtrigraphs $(CFLAGS)
```

Comment:
Drop ```-O2```

Reply:
You're not reading it right. ?= only defines the variable if it is already undefined. Meaning the line gets skipped.

Reply:
To be clear, this is how I got rid of -fkeep-inline-functions , which is not helping the size.